design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -230,14 +230,15 @@ type NegotiableQuoteHistoryEntry {
     changes: NegotiableQuoteHistoryChanges
 }
 
-type CartPrices {
-    initial_total_price: Money
+type NegotiableQuotePrices implements QuotePricesInterface {
```

Comment:
The type return for NegotiableQuote->prices needs to be updated.

Reply:
solved https://github.com/magento/architecture/pull/489/files#diff-7df827613bd49741c4fecea764fff6242c93851ac71b6eb7f870dc6e88f7132aR188